opm/input/eclipse/EclipseState/Tables/PvtxTable.hpp

```diff
@@ -148,14 +246,75 @@ The first row actually corresponds to saturated values.
         }
 
     protected:
-        ColumnSchema m_outerColumnSchema;
-        TableColumn m_outerColumn;
+        /// Table description of primary lookup key.
+        ///
+        /// Typically the composition (Rs, PVTO) or the pressure (Pg, PVTG).
+        ColumnSchema m_outerColumnSchema{};
+
+        /// Primary lookup key values.
+        TableColumn m_outerColumn{};
+
+        /// Table description of under-saturated states.
+        TableSchema m_underSaturatedSchema{};
 
-        TableSchema m_underSaturatedSchema;
-        TableSchema m_saturatedSchema;
-        std::vector< SimpleTable > m_underSaturatedTables;
-        SimpleTable m_saturatedTable;
+        /// Table description of saturated states.
+        TableSchema m_saturatedSchema{};
+
+        /// Under-saturated sub-tables.
+        ///
+        /// One table for each value of the primary lookup key.
+        std::vector<SimpleTable> m_underSaturatedTables{};
+
+        /// Inferred table of saturated states.
+        SimpleTable m_saturatedTable{};
+
+        /// Populate internal data structures from PVTx input table data.
+        ///
+        /// Fills the under-saturated sub-tables and generates the inferred
+        /// "saturated" table for a single PVT region.  Clients--i.e.,
+        /// derived classes--must define the table "schema" members (i.e.,
+        /// m_underSaturatedSchema and m_saturatedSchema) prior to calling
+        /// init().
+        ///
+        /// \param[in] keyword Input PVTx table data.  Typically containing
+        /// PVTO or PVTG (or similar) tables.
+        ///
+        /// \param[in] tableIdx Zero-based region index for which to
+        /// internalise table data.
+        void init(const DeckKeyword& keyword, std::size_t tableIdx);
+
+    private:
+        /// Populate collection of under-saturated tables.
+        ///
+        /// Inserts values into \c m_underSaturatedTables.  This is first
```

Comment:
the

Reply:
> the

Good catch.  I've pushed an update to correct this.